wormhole-connect/src/utils/transferValidation.ts

```diff
@@ -81,8 +81,6 @@ export const validateAmount = (
   if (paymentOption === PaymentOption.MANUAL) return '';
   if (!minAmt) return '';
   if (amount < minAmt) return `Minimum amount is ${minAmt}`;
-  if (amount + minAmt > b)
-    return 'Amount plus estimated fees exceeds the wallet balance';
```

Comment:
Double check, but I think this is just wrong.  It takes fees out of the amount, not in addition